conf/test.config

```diff
@@ -22,4 +22,6 @@ params {
     // Input data
     input  = 'assets/1.0.0/test_samplesheet.csv'
 
+    // Tool options
+    mash_screen_reference_fasta     = 'https://millardlab-inphared.s3.climb.ac.uk/2Jul2023_refseq_genomes.fa.gz'
```

Comment:
will this fasta file be always publicly available? else it will have to move to nf-core/test-datasets

Reply:
Great point, I will update this to use an already existing nf-core/test-dataset